### PR TITLE
Add Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by kinow**
Hi,

Brazilian Portuguese added as per instructions from the Wiki. @osma first time using the `trans_script` script. It changed other files (en, fr, etc), but I removed them. Got that feeling of I-dont-know-what-I-am-doing :slightly_smiling_face: so let me know if I made anything wrong, please.

@brunoalmeida81 let me know if you would like to review this one. Happy to fix things on Transifex and update this PR later.

Cheers
Bruno

### CODE REVIEW

**Comment by osma** [comment]
Wow that was fast @kinow! Looks good to me! 

Should I merge this now or does @brunoalmeida81 want to take a look first?

File: resource/translations/skosmos_pt_BR.po [hunk 1]
```diff
@@ -0,0 +1,770 @@
...
+"X-Poedit-SearchPath-2: model\n"
+"X-Poedit-SourceCharset: utf-8\n"
+
+msgid "Download this vocabulary:"
+msgstr "Baixar este vocabulário:"
+
+#: controller/Controller.php:38
+#: /tmp/cache/587c83c09abaa/4e/4e05e5051b20c875609479922a558d59b8500ae386eaf79bc460e69d8781a753.php:21
+msgid "in_this_language"
+msgstr "em Inglês"
```
Comment (osma) [inline resource/translations/skosmos_pt_BR.po:31]: this should be the equivalent of "in Portuguese" or perhaps more likely "in Brazilian Portuguese" - this is what is shown as the label of the link to change the UI language

Comment (kinow) [reply resource/translations/skosmos_pt_BR.po:31]: Aaahhh

**Comment by brunoalmeida81** [comment]
@kinow @osma Sure, I will take a look.

File: resource/translations/skosmos_pt_BR.po [hunk 2]
```diff
@@ -0,0 +1,770 @@
...
+"X-Poedit-SearchPath-2: model\n"
+"X-Poedit-SourceCharset: utf-8\n"
+
+msgid "Download this vocabulary:"
+msgstr "Baixar este vocabulário:"
+
+#: controller/Controller.php:38
+#: /tmp/cache/587c83c09abaa/4e/4e05e5051b20c875609479922a558d59b8500ae386eaf79bc460e69d8781a753.php:21
+msgid "in_this_language"
+msgstr "em Português"
```
Comment (kinow) [inline resource/translations/skosmos_pt_BR.po:31]: @osma fixed!

**Comment by kinow** [comment]
Thank you @brunoalmeida81 ! I never read anything about semantic web, ontologies, and vocabularies in Portuguese. So most of my translations for certain terms were either based on yours, or after comparing output of priberam, dicionário aurélio, and google translate :sweat_smile: feel free to suggest improvements :+1:

File: resource/translations/skosmos_pt_BR.po [hunk 3]
```diff
@@ -0,0 +1,770 @@
...
+msgstr "Digite seu nome"
+
+#: /tmp/cache/587c83c09abaa/3c/3c4ea3202d0add91bf2080022b707c23115e0beb434024ddb2de27749e6bbe83.php:900
+#: /tmp/cache/587c83c13194c/5c/5c060d2297f64d66beb101ee34831f917ed8d33f97dadeca144430a1561433ec.php:69
+msgid "Error: Requested vocabulary not found!"
+msgstr "Erro: Não foi possível encontrar o vocabulário especificado"
+
+#: /tmp/cache/587c83c09abaa/3c/3c4ea3202d0add91bf2080022b707c23115e0beb434024ddb2de27749e6bbe83.php:890
+msgid "Error: Term \"%term%\" not found in vocabulary!"
+msgstr "Error: Termo \"%term%\" náo foi encontrado no vocabulário~"
```
Comment (brunoalmeida81) [inline resource/translations/skosmos_pt_BR.po:160]: It should be something like:  "Erro: Termo \"%term%\" não foi encontrado no vocabulário!"

File: resource/translations/skosmos_pt_BR.po [hunk 4]
```diff
@@ -0,0 +1,770 @@
...
+msgid "Feedback"
+msgstr "Feedback"
+
+#: /tmp/cache/587c83c09abaa/4a/4a8733799c4f4aa865ece4715881ffcbef92985ed08edc2acd445e7b0c129fab.php:64
+msgid "Feedback has been sent!"
+msgstr "O seu feedback foi enviado com sucesso!"
+
+#: /tmp/cache/587c83c09abaa/d1/d13f81e53ff33f31fff259d66f8ddbf7cfc39d25c6219ff727553feca6d87d37.php:35
+msgid "Group index"
+msgstr "Índice for grupo"
```
Comment (brunoalmeida81) [inline resource/translations/skosmos_pt_BR.po:174]: "Índice de grupo"?

File: resource/translations/skosmos_pt_BR.po [hunk 5]
```diff
@@ -0,0 +1,770 @@
...
+msgid "Search options"
+msgstr "Opções de busca"
+
+#: /tmp/cache/587c83c09abaa/4a/4a8733799c4f4aa865ece4715881ffcbef92985ed08edc2acd445e7b0c129fab.php:161
+msgid "Send feedback"
+msgstr "Enviar feedback"
+
+#: /tmp/cache/587c83c09abaa/f2/f29f57cf31b29541d6f810d43d1f7a07e79cfddcf23678f04612a410c0f0607b.php:51
+msgid "show all # paths"
+msgstr "mostrar todos os # paths"
```
Comment (brunoalmeida81) [inline resource/translations/skosmos_pt_BR.po:248]: Traduzir também "paths". Talvez: "mostrar todos os # caminhos"

File: resource/translations/skosmos_pt_BR.po [hunk 6]
```diff
@@ -0,0 +1,770 @@
...
+msgid "Skosmos version %version%"
+msgstr "Versão do Skosmos %version%"
+
+#: /tmp/cache/587c83c09abaa/9a/9a6d69d72b35f38585477d2f25d50cacf270791aa84e4d523f237566dd428822.php:229
+msgid "Submit search"
+msgstr "Buscar"
+
+#: /tmp/cache/587c83c09abaa/4a/4a8733799c4f4aa865ece4715881ffcbef92985ed08edc2acd445e7b0c129fab.php:68
+msgid "Thank you for your feedback"
+msgstr "Obrigado pelo seu feedback. Nós entraremos em contato o mais breve o possível."
```
Comment (brunoalmeida81) [inline resource/translations/skosmos_pt_BR.po:263]: "... o mais breve possível." (e não "... o mais breve o possível.")

File: resource/translations/skosmos_pt_BR.po (hunk 5 above)
Comment (kinow) [reply resource/translations/skosmos_pt_BR.po:248]: Ah! Caminhos! "Estava na ponta da língua!". Thanks!

File: resource/translations/skosmos_pt_BR.po (hunk 4 above)
Comment (kinow) [reply resource/translations/skosmos_pt_BR.po:174]: Momentary brain glitch.

**Comment by kinow** [comment]
> Just a few suggestions. Nice work!

Thanks for the prompt review @brunoalmeida81 . It was based on your great work.

Pushed (force) to this branch again. You can view only the changes on this link https://github.com/NatLibFi/Skosmos/compare/6bf1b35271c683c29617a533e7f25781fd3d5670..b801478eac8d3c2262ba8a0bc30b274e19756fe0 (or clicking on the **force pushed** in the link that says "kinow force-pushed the (...)" above).

Thanks!

**Comment by osma** [comment]
All happy now? Should I merge this?

**Comment by brunoalmeida81** [comment]
@osma @kinow Yes, its fine by me.

**Comment by osma** [comment]
Thanks again, both!

**Comment by kinow** [comment]
Thanks @osma and @brunoalmeida81 !

**Pull request merged**